stackdriver_debugger.c

```diff
@@ -64,6 +69,9 @@ static zend_function_entry stackdriver_debugger_functions[] = {
     PHP_FE(stackdriver_debugger_add_logpoint, arginfo_stackdriver_debugger_add_logpoint)
     PHP_FE(stackdriver_debugger_list_logpoints, NULL)
     PHP_FE(stackdriver_debugger_valid_statement, arginfo_stackdriver_debugger_valid_statement)
+    PHP_FE(stackdriver_debugger_opcache_enabled, NULL)
```

Comment:
Perhaps, I should remove these helper functions that are provided to PHP user-space?

Reply:
Ah good point. Exposed functions will become public interface. Less is better. Feel free to do it with another PR, or do it within this PR